src/js/components/receiving/PartialReceivingPage.jsx

```diff
@@ -469,6 +536,9 @@ class PartialReceivingPage extends Component {
             locationId: this.props.locationId,
             saveDisabled: !isAnyItemSelected(this.props.formValues.containers),
             shipmentReceived: this.props.formValues.shipmentStatus === 'RECEIVED',
+            exportTemplate: this.exportTemplate,
+            formValues: this.props.formValues,
```

Comment:
you don't need to pass the formValues as a parameter here, you can use it like it's done in onSave function